src/main/java/frc/robot/subsystems/module/Module.java

```diff
@@ -42,7 +44,7 @@ public class Module extends SubsystemBase {
 
     protected boolean stateDeadband = true;
 
-    SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(DriveConstants.DRIVE_KS, DriveConstants.DRIVE_KV, DriveConstants.DRIVE_KA);
+    SimpleMotorFeedforward feedforward;
     
     final VelocityVoltage m_VelocityVoltage = new VelocityVoltage(0);
```

Comment:
Even though you didn't add them in this PR, all of the module instance variables should be private. Even though ModuleSim might extend it, it doesn't need to access any of these. It can use the get/set methods if needed.

Reply:
made them private